src/components/Header/Header.test.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {
+  cleanup,
+  fireEvent,
```

Comment:
remove unused imports

Reply:
fixed in 7e6c6003d60350fc9b2eb63ec1aecf1599f3dfdc